src/Orc.Controls/Controls/LogViewer/Models/Action.cs

```diff
@@ -0,0 +1,16 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="Action.cs" company="WildGums">
+//   Copyright (c) 2008 - 2018 WildGums. All rights reserved.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+
+namespace Orc.Controls.Models
```

Comment:
Please use Orc.Controls namespace and give it a more meaningful name (e.g. LogFilterAction)